TESTNET.md

```diff
@@ -0,0 +1,36 @@
+# How to set up a corda k8s node on the corda testnet network
+
+Azure steps :
+- create an AKS cluster
+- create a storage account with 3 file share:
+  - node-storage (2Gb)
+  - bridge-storage (1Gb)
+  - float-storage (1Gb)
+- create an ACR (docker registry)
+- create 2 public IP adresses in the same RG as AKS (starts with MC_)
+  - p2p-ip
+  - rpc-ip
+- fill `helm/values.yaml` with appropriate values from above
+
+Repo steps :
+- Do docker steps (docker directory)
+- Register on corda marketplace : https://marketplace.r3.com/register
```

Comment:
The following block should be integrated to the CHECKLIST.md, but with a heading for Testnet configuration